probe_scraper/parsers/scalars.py

```diff
@@ -7,11 +7,18 @@
 
 
 def extract_scalar_data(s):
+
+    # External scalars.yaml files have release/prerelease, not opt-in/opt-out
+    try:
+        optout = s.dataset.endswith('_OPTOUT')
+    except KeyError:
+        optout = s._definition.get('collect_on_channels', 'prerelease') == 'release'
```

Comment:
Why not add this as a property to the parser that encapsulates this?
(Hm, did i comment on this before?)

Reply:
You did, this would be a dependency on `parse_scalars.py`.